controllers.js

```diff
@@ -70,4 +70,14 @@ function editVotesByArticleId(req, res, next) {
     .catch((err) => next(err))
 }
 
-module.exports = { getTopics, getDescription, getArticleById, getArticles, getCommentsByArticleId, createComment, editVotesByArticleId }
\ No newline at end of file
+
+function deleteCommentById(req, res, next) {
+  const commentId = req.params.comment_id;
+  removeCommentById(commentId)
+    .then(() => {
+      res.status(204).send();
```

Comment:
Exactly right - when we delete we don't send a body back, and instead just send a 204 as confirmation that it worked